Added some filters in listview template for jobby theme compatibility

mistergiri opened the pull request.

NomadDevs commented:
I dont see why we need these changes, please expalin them all, and if we do need them then we probably need them also in widget listing view

mistergiri commented:
I have added 1 filter and 2 actions. 

```geodir_show_listing_post_excerpt``` filter for disabling excerpt.

I wanted to add one more column after the content column for location. So I have added this action ```geodir_after_listing_content``` 

```geodir_before_review_html``` added to insert meta fields before review stars.

With those filter and action in place, it looks like this.

http://www.awesomescreenshot.com/image/611444/568c4a52fdfbc8cce48311a328362062

Those filters are added for jobby listing page compatibility. So I don't think we need those hooks in widgets too.
